src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -288,8 +291,11 @@ export default function Page() {
                     </> : <></>}
                 <DateSelectorCard onDateChange={updateDateRange} date={selectedDate} />
                 
+                
             </HStack>
             {useMemo(() => renderWarning(), [selectedDate])}
+        <div>
+          <fieldset disabled={disabled}>
```

Comment:
❓Why use fieldset here?

Reply:
I just used it to group the required fields together so I could disable them all at once if required